tests/resources/test_resource_utils.py

```diff
@@ -14,6 +14,13 @@
 )
 
 
+@pytest.fixture(autouse=True)
+def reset_gnomad_public_resource_configuration():
```

Comment:
naive question, why do you need this?

Reply:
The tests below set `gnomad_public_resource_configuration` to different things for testing, and this sets it back to None so that the other tests can run correctly. I was very confused for awhile when the tests I added kept coming back with an error about the gencode resource not being found in "gs://my-bucket/gnomad-resources"